### PR TITLE
fix: ReadMe docs (tests)

### PR DESCRIPTION
**Pull request by ggilberth**
fix: readme docs - fix action so payload in expect is correct, as per createActions-test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ const { actionOne, actionTwo, actionThree } = createActions({
 
   // array form
   ACTION_TWO: [
-    (first) => first,               // payload
+    (first) => [first],               // payload
```
Comment (timche) [inline README.md]: Align `// payload` with `// meta` 😄

Comment (ggilberth) [inline README.md]: Done! 😅

**Comment by timche** [comment]
Btw, it looks like you haven't set your git properly. Normally your GitHub username should be displayed as commit author [here](https://github.com/acdlite/redux-actions/pull/180/commits), but it doesn't.

Make sure `git config --get user.email` (inside the repository) is your GitHub email.

**Comment by ggilberth** [comment]
Thanks for letting me know. Fixed it.

**Comment by timche** [comment]
You're welcome :) You can also set it globally, so you don't need to do it every time: https://git-scm.com/book/tr/v2/Customizing-Git-Git-Configuration

**Comment by timche** [comment]
Merged, thanks for your contribution! :)

**Pull request merged**